### DISCUSSION TITLE
Library list during compile action

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1901] [IBM i] [answered]
I believe I have stumbled onto a bug. This recently prevented me from a successful compile because the library PROGRAMMER had an old version of a file (circa 1994) which didn't have many new fields. My library list was set correctly in my session (upper left), which has the current version of the file in library DISK. But when the run action ran, it does correctly show my current library, but the library list is reversed.
![image](https://github.com/codefori/vscode-ibmi/assets/132368707/1d8ad886-6746-440e-8292-f0ccfd1e8f8a)

This caused the compile to fail because it took the 1994 version (which I've since deleted, lol) out of the PROGRAMMER library instead of the intended DISK library.

**Comment by chrjorgensen** [comment, marked as answer]
Hi @Gregory1701 

This is an error that has been fixed in the latest release v2.8.0 of Code for IBM i, just published.

Please upgrade and verify that your problem has been solved.  😊

**Reply by Gregory1701** [reply]
Thanks @chrjorgensen , will do!

**Reply by Gregory1701** [reply]
Much better, thank you!
